File: aoj/volume3/n0359/Main.java
import java.util.Scanner;

public class Main {

    private static String[] DAYS = {"thu", "fri", "sat", "sun", "mon", "tue", "wed"};

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        String day = DAYS[sc.nextInt() % 7];

        System.out.println(day);

    }

}

File: aoj/volume3/n0359/Main.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {

    const string DAYS[7] = {"thu", "fri" , "sat", "sun", "mon", "tue", "wed"};

    int x;
    cin >> x;

    string day = DAYS[x % 7];
    cout << day << endl;

    return 0;

}
